package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@open-rights-exchange/orejs",
-  "version": "2.3.0",
+  "version": "2.3.1",
```

Comment:
This version adds new functionality so it should be 2.4.0

Reply:
done